10.cpp

```diff
@@ -1,84 +1,30 @@
-#include<iostream>
-using namespace std;
-int main()
-{
-	int n, m, i, a[30], b[30], sum[30], x, l;
-
-	//initialising sum array as 0
-	for (i = 0; i < 30; i++)
-	{
-		sum[i] = 0;
-		a[i] = 0;
-		b[i] = 0;
-	}
-
-	//input array 1
-	cin >> n;
-	for (i = 0; i < n; i++)
-		cin >> a[i];
-
-	//input array 2
-	cin >> m;
-	for (i = 0; i < m; i++)
-		cin >> b[i];
-
-	//array resizing
-	if (n > m)
-	{
-		l = n - m;
-		for (i = m; i >= 0; i--)
-			b[i + l] = b[i];
-		for (i = 0; i < l; i++)
-			b[i] = 0;
-	}
+# include <iostream>
```

Comment:
**`#include <bits/stdc++.h>`** is commonly used. It helps save time to not think about what functions/objects are inside which header.

If during compilation any errors occur, you may add the necessary headers. Otherwise **`#include <bits/stdc++.h>`** works.